src/store/dicom-web/dicom-web-store.ts

```diff
@@ -203,6 +203,7 @@ export const useDicomWebStore = defineStore('dicom-web', () => {
         state: 'Done',
       };
     } catch (error) {
+      console.error(error);
```

Comment:
Do you want to keep this, or was just added during debugging?

Reply:
Good check, did want to keep actualy.